### PR TITLE
Fix minor issue found during static code scan

### PR DESCRIPTION
**Pull request by DavidZagury**
Fix issue of using uninitialized value on unit tests.

### CODE REVIEW

**Review by prsunny — DISMISSED** [review]

**Comment by keboliu** [comment]
@prsunny @qiluo-msft  would you be able to merge this PR?

File: common/ipprefix.h [hunk 1]
```diff
@@ -76,7 +76,8 @@ class IpPrefix
                 {
                     assert(mid >= 0 && mid < 16);
                     ipa.ip_addr.ipv6_addr[mid] = (uint8_t)(0xFF << (8 - bits));
-                    memset(ipa.ip_addr.ipv6_addr + mid + 1, 0, 16 - mid - 1);
+                    if (mid < 15)
```
Comment (qiluo-msft) [inline common/ipprefix.h:79]: >if (mid < 15) [](http://example.com/codeflow?start=20&length=13)

Wondering who is complaining? the compiler?

**Comment by liat-grozovik** [comment]
@qiluo-msft any further comments or can we merge this one?

File: common/ipprefix.h (hunk 1 above)
Comment (qiluo-msft) [reply common/ipprefix.h:79]: This error does not make sense, it indicates the limitation of scanner/linter you are using.
I suggest not to sacrifice code concision to workaround scanner’s issue. If you like, you can add a code comment to suppress scanner negative alarm.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**